astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -318,7 +320,9 @@ def validate(cls, value):
         return parameters
 
     @classmethod
-    def register(cls, name: str, parameters: dict, references=None, **meta: dict):
+    def register(
+        cls, name: str, parameters: dict, references=None, **meta: dict
```

Comment:
I guess ruff's rule set uses the more lax version of mypy none-type annotation?
https://mypy.readthedocs.io/en/stable/config_file.html#confval-implicit_optional

Reply:
This does violate `ANN001`, but I'm not planning to address that rule here.